Source/Beam Simulator/BeamSimulator.py

```diff
@@ -0,0 +1,57 @@
+import scipy as sp
+import numpy as np
+
+def ClassicalBeam(V: int, LM: int, RM: int):
+    '''
+    ClassicalBeam takes the inputs of the SPARC apparatus and simulates the trajectory of the electron beam using
+    non-relativistic models of physics
+
+    :param V: Voltage input
+    :param LM: Left Magnet Position
+    :param RM:  Right Magnet Position
+    :return: np.array
+    '''
+
+    # Define charge q_e and rest mass m_e of an electron
+    q_e = 1.60217663e-19
```

Comment:
```suggestion
    q_e = const.pi
```
Looks like there's a [`scipy.constants`](https://docs.scipy.org/doc/scipy/reference/constants.html#) submodule, can we use it instead? TODO: Also use it for the electron mass, charge, and speed of light: https://docs.scipy.org/doc/scipy/reference/constants.html#physical-constants.

Reply:
I'm fine with using a submodule in scipy, but why pi?

Reply:
Lul I didn't mean to type out pi but you get the idea. Wiki sounds good to me, I suggested adding the tex file so others could contribute.

Reply:
I'll take a look at the scipy module. Also, the PDF is just compiled LaTeX from overleaf which I think I already shared. I'll post it again in teams.

Reply:
> Looks like there's a [`scipy.constants`](https://docs.scipy.org/doc/scipy/reference/constants.html#) submodule, can we use it instead?

I've taken a look. We can do this but is there any specific advantages of doing so?

Reply:
I don't know if it has any effect on perf. However, I generally choose to use library APIs since they exist for others to use and come with additional functionality (scipy.constants also has a units API). Feel free to leave them as is if you don't see any benefits.

Reply:
I've added them because they have more precision than the figures I used and the scipy.constants API might come in handy later